website/docs/guides/visual-editor-qs.md

```diff
@@ -0,0 +1,287 @@
+---
+title: "Quickstart for the dbt Cloud Visual Editor"
+id: "visual-editor"
+level: 'Beginner'
+icon: 'dbt'
+hide_table_of_contents: true
+tags: ['Visual Editor','Analyist', 'dbt Cloud','model']
+recently_updated: true
+---
+
+<div style={{maxWidth: '900px'}}>
+
+## Introduction
+
+:::note beta feature
+
+The Visual Editor is currently in a limited beta. [Contact us](https://www.getdbt.com/contact/) if you're interested in joining it. Features currently in the beta are subject to change or removal.
+
+:::
+
+The dbt Cloud Visual Editor offers a quick and straightforward way for anyone to build analytics models, no background in analytics engineering is required! In this guide, you will learn about:
+
+- Accessing the Visual Editor and creating a new model
+- How to navigate the interface
+- How to build a model using operators
+- How to commit your changes to Git
+- Locating your Visual Editor model and data
+
+## Prerequisites
+
+To use the Visual Editor, you must meet the following prerequisites:
+
+- Be on an [Enterprise dbt Cloud account](/faqs/Accounts/cloud-upgrade-instructions#enterprise-plan).
+- Your account must have the following configured:
+    - A data warehouse connection
+    - Integration with a Git provider
+- Source models for the Visual Editor must have been run at least once
+- You must have a `developer` license
+- You must have credentials configured for your data warehouse and Git provider in the **Your profile** section of the **Account settings**. 
+
+:::note
+
+The examples in this guide use the [Jaffle Shop](https://github.com/dbt-labs/jaffle-shop) GitHub repo sample project. You can use your own data, but the Jaffle Shop offers a full-featured project useful for testing dbt features. Ask your dbt Cloud administrator about importing it to a project in your environment. They must also execute `dbt run` on the Jaffle Shop project in your `Production` environment before you begin, or you will be unable to reference the source models.
+
+:::
+
+## Access the Visual Editor
+
+To access the Visual Editor:
+
+1. Click **Develop** from the main menu. If you do not see the **Develop** option, ensure you have selected a **Project** from the menu.
+2. Click **Visual Editor**.
+
+<Lightbox src="/img/docs/dbt-cloud/visual-editor/ve-main-menu.png" width="90%" title="Visual Editor in the main menu."/>
+
+3. From the right side, click **Get started** and then click **Create new model**.
+
+<Lightbox src="/img/docs/dbt-cloud/visual-editor/ve-create-new-model.png" width="90%" title="Create a new model from the Visual Editor landing page."/>
```

Comment:
For this image, you might consider only capturing the white space. Thinking about longer-term image maintenance.